samples/test/metrics_visualization_v2.py

```diff
@@ -113,9 +115,27 @@ def iris_sgdclassifier(test_samples_fraction: float, metrics: Output[Classificat
         confusion_matrix(train_y, predictions).tolist() # .tolist() to convert np array to list.
     )
 
+@component(
+    kfp_package_path='git+https://github.com/kubeflow/pipelines#egg=kfp&subdirectory=sdk/python'
```

Comment:
Sorry, I was suggesting this for local testing. 
For tests, we should still use that environment variable which will install KFP under a commit hash.

Reply:
Sounds good, fixed.